src/model/inventory/Ingredient.java

```diff
@@ -0,0 +1,11 @@
+package model.inventory;
+
+/**
+ * This class will be use in the inventory and wil be use in products for the preparation
```

Comment:
typo, instead couldbe: `This class will be used by the inventory and will be used and final product preparation`.

Reply:
Javadoc Fixed!